example/src/classes/view/main/main-view.js

```diff
@@ -0,0 +1,33 @@
+import ElementBuilder from '../../util/element/element-builder';
+import View from '../view';
+
+const CssClasses = {
+    MAIN: 'main',
+};
+
+export default class MainView extends View {
+    constructor() {
+        super();
+        this.htmlElement = this.createView();
```

Comment:
Base class should do this work automatically. We would just pass needed data to `super`.